### ISSUE TITLE
[Question/Request] Structures

### ISSUE THREAD

**Issue by starterdragon** [Ad5001/RPCompanies#2]
Can you make it to have a feature like EconomyProperty , so that players can buy PreBuilt structures or make one with Empty lots.

If ($this->exist($issue))
{
return true;
}
else
{
$dev->reply();
}

**Comment by Ad5001** [comment]
Well, I will see because people would ask me to make them customisable. And since I don't know anything about the schematics format...

**Comment by starterdragon** [comment]
no, like for example. the server has a building and an op will set point 1 to point 2 then use command like /rpc forsale thisbuilding 500000

then a player can use /rpc buy thisbuilding .. or a sign which can be placed anywhere like this
[RPCompanies]
forsale
thisbuilding
[and it will automatically set price here]

then the sign will be automatically break when bought

**Comment by Ad5001** [comment]
Ohh !
This gives me an another idea !
A builder company that would make buildings !
